### DISCUSSION TITLE
Personal Access Token -- Used in last X

### DISCUSSION THREAD

**Discussion by match-caseydaniell** [community/community#24531] [Enterprise]
<p>Using: Github Enterprise Onprem</p>
<p>I am looking at a build user that we use to perform some automated task, and noticed some of the users Personal Access Tokens “Last used within the last 2 years” vs “Last used within the last week”</p>
<p>Is this really saying that this token hasn’t performed any actions in over a year, and under 2 years?</p>
<p>(If so, I want to delete it, but wanted to verify)</p>

**Comment by francisfuzz** [comment, marked as answer]
<p>Hey <a class="mention" href="/caseydaniell">@caseydaniell</a>––welcome back and raising this question here!</p>
<aside class="quote no-group" data-username="caseydaniell" data-post="1" data-topic="116857">
<div class="title">
<div class="quote-controls"></div>
 caseydaniell:</div>
<blockquote>
<p>Is this really saying that this token hasn’t performed any actions in over a year, and under 2 years?</p>
<p>(If so, I want to delete it, but wanted to verify)</p>
</blockquote>
</aside>
<p>That’s spot on. :ok_hand: To confirm:</p>
<ul>
<li>“Last used within the last week” means that the token was last used sometime between now and the last 7 days.</li>
<li>“Last used within the last 2 years” means that it was used over a year ago, but less than 2 years.</li>
</ul>
<p>We hope this clarification helps!</p>
